Firestore/Example/Tests/Util/FSTHelpers.h

```diff
@@ -21,7 +21,12 @@
 #import "Firestore/Source/Model/FSTDocumentDictionary.h"
 #import "Firestore/Source/Model/FSTDocumentKeySet.h"
 
+@class FIRCollectionReference;
+@class FIRDocumentReference;
+@class FIRDocumentSnapshot;
+@class FIRFirestore;
 @class FIRGeoPoint;
+@class FIRQuerySnapshot;
```

Comment:
I think all of these added FIR* forward declarations are no longer necessary?

Reply:
done